packages/puppeteer-core/src/cdp/BrowserConnector.ts

```diff
@@ -29,6 +29,8 @@ import {isErrorLike} from '../util/ErrorLike.js';
 import {CdpBrowser} from './Browser.js';
 import {Connection} from './Connection.js';
 import type {ConnectOptions} from './ConnectOptions.js';
+import {BidiBrowser} from '../bidi/Browser';
```

Comment:
We should be doing this part with dynamic import similar to `launch`.

Reply:
Rolled back that part and focused only on the tests in this PR